scripts/pre_push_hook.py

```diff
@@ -598,8 +362,13 @@ def main(args: Optional[List[str]] = None) -> None:
             frontend_status = 0
             ci_check_status = 0
             if does_diff_include_js_or_ts_files(files_to_lint):
+                js_or_ts_files = git_changes_utils.get_js_or_ts_files_from_diff(
+                    files_to_lint)
```

Comment:
Seems like this could be simplified to getting the js/ts files from the diff and then checking whether the list is empty

Reply:
Done.